docs/learn-staking.md

```diff
@@ -74,74 +78,72 @@ There are three different accounts for managing your funds: `Stash`, `Controller
 
 ![staking](assets/NPoS/staking-keys.png)
 
-- **Stash:** This account holds funds bonded for participation, but delegates its staking and governance functions to controller and proxy keys. As a result, you may actively participate with a stash key kept in a cold wallet, meaning it stays offline all the time, possibly sharded in bank vaults. After unbonding, users must wait a certain amount of time in order to access the locked funds (600 blocks at the time of writing).
-- **Controller** This account controls its stash account's nomination of validator nodes, or authorizes operation of your own validator, switching between validating, nominating and idle. It only needs enough funds to post transactions when actions are taken.
-- **Proxy** This account participates in governance on behalf of its stash account. Again, it only needs enough funds to post vote transactions.
-- **Session** Session keys are not account keys, but instead consist of several different key types used by validator nodes for different functions. A validator operator first certifies their session keys with their controller key. We recommend handling session keys using only your node's RPC interface because if session keys exist elsewhere then you might equivocate and be slashed. We still support the legacy `--key` parameter for testnets like Alexander.
+* **Stash:** This account holds funds bonded for participation, but delegates its staking and governance functions to controller and proxy keys. As a result, you may actively participate with a stash key kept in a cold wallet, meaning it stays offline all the time, possibly secured by a multi-signature wallet with member keys in different bank vaults. After unbonding, users must wait a certain amount of time in order to access the locked funds (600 blocks at the time of writing).
+* **Controller** This account controls its stash account's nomination of validator nodes, or authorizes operation of your own validator, switching between validating, nominating and idle. It only needs enough funds to post transactions when actions are taken.
+* **Proxy** This account participates in governance on behalf of its stash account. Again, it only needs enough funds to post vote transactions.
+* **Session** Session keys are not account keys, but instead consist of several different key types used by validator nodes for different functions. A validator operator first certifies their session keys with their controller key. We recommend handling session keys using only your node's RPC interface because if session keys exist elsewhere then you might equivocate and be slashed. We still support the legacy `--key` parameter for testnets like Alexander.
```

Comment:
Although stated in the first sentence, session keys are not accounts. So maybe they shouldn't be under the "Accounts" section.

Reply:
I agree.

Reply:
Where then?

Reply:
I think we could just say in the "Controller" section that this account sets the Session keys and link to the [keys page](https://wiki.polkadot.network/docs/en/learn-keys#session-keys). The image above should also be updated --> remove BLS, maybe have Stash point to Controller and Proxy. I'd also just remove Session keys, say that the Controller sets the Session keys.

Should also fix this sentence above the image: "There are three different accounts for managing your funds: Stash, Controller and Session accounts."

Reply:
Thinking further, this page warrants a link to Session keys, but the rest of it is very high level, like how slashing/rewards work, NPoS, nominator vs validator, etc. Session keys are a low level concern for validators when they are deploying their system, but are not at all related to this document, which is "How does staking work and why would I want to do it?"

Reply:
I propose merging this as is, because the content we're talking about here has been in there for a while now anyway, just so the fixes are in. Then we can do these clarifications in a separate PR.

Reply:
I see what you're saying but it needs to happen fast. The risk is that it goes on the "later list" and actually never happens. This image above this list has actually caused confusion with wallet implementation teams. Reworking this page should be high on our priority list.

Reply:
Image and text updated, please check